src/mss/MssHandler.js

```diff
@@ -106,9 +104,14 @@ function MssHandler(config) {
         mssFragmentProcessor.processMoof(e);
     }
 
+    function registerEvents() {
+        eventBus.on(Events.INIT_REQUESTED, onInitializationRequested, instance, EventBus.EVENT_PRIORITY_HIGH);
+        eventBus.on(MediaPlayerEvents.FRAGMENT_LOADING_COMPLETED, onSegmentMediaLoaded, instance, EventBus.EVENT_PRIORITY_HIGH);
+    }
+
     function reset() {
         eventBus.off(Events.INIT_REQUESTED, onInitializationRequested, this);
-        eventBus.off(Events.FRAGMENT_LOADING_COMPLETED, onSegmentMediaLoaded, this);
+        eventBus.off(MediaPlayerEvents.FRAGMENT_LOADING_COMPLETED, onSegmentMediaLoaded, this);
```

Comment:
This is not following the systems in place for even aggregation. You should ONLY reference Events object inside the client and annotate with public_ which events you want for external access as well which will be pushed to MediaPlayerEvents class on init..